src/GafferScene/SetFilter.cpp

```diff
@@ -121,6 +150,17 @@ void SetFilter::hashMatch( const ScenePlug *scene, const Gaffer::Context *contex
 	h.append( scene->setHash( setPlug()->getValue() ) );
 }
 
+void SetFilter::compute( Gaffer::ValuePlug *output, const Gaffer::Context *context ) const
```

Comment:
Put this just below `SetFilter::hash()`, so it's easy to verify the two functions match. Making sure that hash/compute have matching behaviour is one of the most irksome things about coding for Gaffer, and keeping them together does help a bit.